### PR TITLE
Enable Node Save

### PR DESCRIPTION
**Pull request by dawehner**
This PR is blocked on having a author field and a image upload field.

When committing we should add @e0ipso as contributor, given he helped with fixing the POST

### CODE REVIEW

File: src/components/05_pages/NodeForm/index.js [hunk 1]
```diff
@@ -15,8 +20,17 @@ schemaType = PropTypes.shape({
   properties: PropTypes.objectOf(lazyFunction(lazySchemaType)),
 }).isRequired;
 
+const styles = () => ({
+  container: {
+    display: 'flex',
```
Comment (martinfrances107) [inline src/components/05_pages/NodeForm/index.js]: There is a little bug hiding here 

it is similar to #202 - single quotes surrounding text properties of CSS styles is invalid.
This applies to 'flex', 'wrap' and 'column'

I hope this helps.

**Comment by dawehner** [comment]
Wait but isn't this one actual JS?

**Comment by martinfrances107** [comment]
> Wait but isn't this one actual JS?

Sorry the other example I quoted was using string template back-ticks.. so my comment is just noise.

**Comment by dawehner** [comment]
Yes we do have issues for those. I know Francesco was working on it

**Comment by dawehner** [comment]
I created a new followup: https://github.com/jsdrupal/drupal-admin-ui/issues/265

**Pull request merged**